examples/02-HFSS/HFSS_eigenmode.py

```diff
@@ -0,0 +1,147 @@
+
+###############################################################################
+# Launch HFSS
+# ~~~~~~~~~~~
+# Launch HFSS 2023 R1 in graphical mode.
+
+hfss = pyaedt.Hfss(projectname=project_path, non_graphical=non_graphical)
+
+###############################################################################
+# Input Parameters for Eigenmode Solver
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The geometry and material should be already set. The anaylses will be generated by the code. Number of modes during each analysis, max allowed number is 20.
+# Entering a number higher than 10 might need long simulation time as the
+# eigenmode solver needs to converge on modes. fmin is the lowest frequency of interest, fmax highest frequesncy of interest.
+# parameter limit is the value, which based on it is decided which modes shall be ignored.
+
+num_modes = 6
+fmin = 1
+fmax = 2
+next_fmin = fmin
+setup_nr = 1
+
+limit = 10
+resonance = {}
+
+###############################################################################
+# Finding the modes
+# ~~~~~~~~~~~~~~~~~
+# The following cell is a function and if called creates an eigenmode setup and solves it.
+# After solve each mode with its corresponding real frequency and quality factor will be saved for further processing.
+
+def find_resonance():
+    #setup creation
+    next_min_freq = str(next_fmin) + " GHz"
+    setup_name = "em_setup" + str(setup_nr)
+    setup = hfss.create_setup(setup_name)
+    setup.props['MinimumFrequency'] = next_min_freq
+    setup.props['NumModes'] = num_modes
+    setup.props['ConvergeOnRealFreq'] = True
+    setup.props['MaximumPasses'] = 10
+    setup.props['MinimumPasses'] = 3
+    setup.props['MaxDeltaFreq'] = 5
+    #analyzing the eigenmode setup
+    hfss.analyze_setup(setup_name, num_cores=8,use_auto_settings=True)
+    #getting the Q and real frequency of each mode
+    eigen_q = hfss.post.available_report_quantities(quantities_category="Eigen Q")
+    eigen_mode =  hfss.post.available_report_quantities()
+    data = {}
+    cont = 0
+    for i in eigen_mode:
+        eigen_q_value = hfss.post.get_solution_data(expressions=eigen_q[cont], setup_sweep_name=setup_name+' : LastAdaptive', report_category = "Eigenmode")
+        eigen_mode_value = hfss.post.get_solution_data(expressions=eigen_mode[cont], setup_sweep_name=setup_name+' : LastAdaptive', report_category = "Eigenmode")
+        data[cont] = [eigen_q_value.data_real()[0], eigen_mode_value.data_real()[0]]
+        cont += 1
+
+    print(data)
+    return data
+
+###############################################################################
+# Automation
+# ~~~~~~~~~~
+# Running the next cell will call the resonance function and save only those modes with a Q higher than the defined
+# limit. The find_resonance function will be called until the complete frequency range is covered.
+# At the end the physical modes in the whole frequency range will be reported.
```

Comment:
```suggestion
# Running the next cell calls the resonance function and saves only those modes with a Q higher than the defined
# limit. The ``find_resonance`` function is called until the complete frequency range is covered.
# When the automation ends, the physical modes in the whole frequency range are reported.
```
We want to avoid future tense.